webpack.config.js

```diff
@@ -31,43 +30,47 @@ module.exports = (env, argv) => { // eslint-disable-line no-undef
     }
 
     const webpack = require(path.resolve(nodeModulesPath, 'webpack'));
-    const CleanObsoleteChunksPlugin = require(path.resolve(nodeModulesPath, 'webpack-clean-obsolete-chunks'));
-    const CleanWebpackPlugin = require(path.resolve(nodeModulesPath, 'clean-webpack-plugin')).CleanWebpackPlugin;
     const ManifestPlugin = require(path.resolve(nodeModulesPath, 'webpack-manifest-plugin')).WebpackManifestPlugin;
     const MiniCssExtractPlugin = require(path.resolve(nodeModulesPath, 'mini-css-extract-plugin'));
-    const OptimizeCssAssetsPlugin = require(path.resolve(nodeModulesPath, 'optimize-css-assets-webpack-plugin'));
+    const CssMinimizerPlugin = require(path.resolve(nodeModulesPath, 'css-minimizer-webpack-plugin'));
     const {styles} = require(path.resolve(nodeModulesPath, '@ckeditor/ckeditor5-dev-utils'));
 
     return {
         entry: [path.resolve(__dirname, 'index.js')], // eslint-disable-line no-undef
         output: {
-            path: path.resolve(projectRootPath, publicDir),
-            filename: outputPath + '/[name].[chunkhash].js',
-            publicPath,
+            clean: true,
+            path: path.resolve(projectRootPath, publicDir, outputPath),
+            filename: '[name].[chunkhash].js',
         },
         stats: 'minimal',
         performance: {
             hints: false,
         },
         devtool: argv.mode === 'development' ? 'eval-source-map' : 'source-map',
```

Comment:
the build process works for me and the result looked good during my testing 🥳 
unfortunately, it looks like our `npm run watch` task does not work anymore. right now, i see the changes only after restarting the watch task

Reply:
the output of my console indicates that the change was detected, but no changes are written to the filesystem if i change the sulu source code

Reply:
okay i suspect that this is somehow caused by the location of the code of Sulu 😢 if i change the code of the `assets/admin/app.js` file, the watch task works as expected. maybe files are not reloaded if they are located within the `node_modules` folder?